com.microsoft.mrtk.input/Visualizers/HandJointVisualizer/HandJointVisualizer.cs

```diff
@@ -12,8 +14,8 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// <summary>
     /// Basic hand joint visualizer that draws an instanced mesh on each hand joint.
     /// </summary>
-    [AddComponentMenu("MRTK/Input/Hand Visualizer")]
-    public class HandVisualizer : MonoBehaviour
+    [AddComponentMenu("Scripts/Microsoft/MRTK/Hands/Hand Joint Visualizer")]
+    public class HandJointVisualizer : MonoBehaviour
```

Comment:
Please make sure this is well documented:)

Reply:
Documented!